core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -432,13 +440,18 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
       logger.warn("""No "path.separator" setting found. Using default value ":" """)
       ":"
     }
-
+    val debugPort = nextDebugPort()
+    logger.info(s"listening for debugging '$pid' at 'localhost:$debugPort'")
     // Using file://localhost/path instead of file:///path or even file://path (as it was originally)
     // appears to be necessary for runLocal to work on both Windows and real systems.
     val forkOptions = ForkOptions()
       .withOutputStrategy(OutputStrategy.LoggedOutput(logger))
       .withConnectInput(false)
-      .withRunJVMOptions(Vector(s"-Dlog4j.configuration=file:///${log4JConfigFile.toFile.getAbsolutePath}"))
+      .withRunJVMOptions(
+        Vector(
+          s"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,quiet=y,address=localhost:$debugPort -Dlog4j.configuration=file:///${log4JConfigFile.toFile.getAbsolutePath}"
```

Comment:
I "think" that it should be optional, debugging java agent affect performance and how the runtime works

Reply:
I thought about it but I think performance is not a concern when running in local. Not sure about the runtime you mention though. What do you have in mind? 
In any case, not a strong opinion here.

Reply:
I'm also thinking that, for the user, is simpler. I believe that UX should be a higher priority.

Reply:
I do like:
> if you really wanted to you could have a setting debug which is on by default.

but mostly wanted to rise this conversation/awareness 😄  so, this is resolved 👍